.github/workflows/publish-setup.yml

```
@@ -0,0 +1,60 @@
+name: setup - Publish Image
+
+on: 
+  release:
+    types: [published]
+
+# Ensure only the most recent workflow runs.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  publish_to_ghcr:
+    name: Publish image to ghcr.io
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+
+      - name: Lowercase repo owner
+        id: lowercase_repo_owner
+        uses: ASzc/change-string-case-action@v1
+        with:
+          string: ${{ github.repository_owner }}
+
+      - name: Setup Image Metadata
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          # list of Docker images to use as base name for tags
+          images: |
+            ghcr.io/${{ steps.lowercase_repo_owner.outputs.lowercase }}/proxy-mediator
```

Name collision with the core image here. Should probably be something like:

```suggestion
            ghcr.io/${{ steps.lowercase_repo_owner.outputs.lowercase }}/proxy-mediator-setup
```